src/ApiPort.VisualStudio.Common/Analyze/FileListAnalyzer.cs

```diff
@@ -20,7 +20,7 @@ public FileListAnalyzer(IVsApiPortAnalyzer analyzer, IFileWriter reportWriter)
 
         public async Task AnalyzeProjectAsync(IEnumerable<string> inputAssemblyPaths)
         {
-            await _analyzer.WriteAnalysisReportsAsync(inputAssemblyPaths, _reportWriter, false).ConfigureAwait(false);
+            await _analyzer.WriteAnalysisReportsAsync(inputAssemblyPaths, null, _reportWriter, false).ConfigureAwait(false);
```

Comment:
I don't see anywhere in this solution where this `null` would be a problem today, but I think an empty enumerable would be safer.

Reply:
Fixed.